### ISSUE TITLE
Estimate time spent between commits to calculate a working hour metric

### ISSUE THREAD

**Issue by Ly0n** [protontypes/openselery#37]
Hey,
we could calculate the hours spend on a pull request and add this to the money distribution metric
You can here find a simple algorithm to cluster the hours between commits.
https://github.com/kimmobrunfeldt/git-hours

**Comment by kikass13** [comment]
sounds awful :D

**Comment by Ly0n** [comment]
> sounds awful :D

Yes it does, but it is one way to evaluate the workload. I would not give that metric a high wight for programming but it could be more important if you want to finance contributor of a blog.
